docs/source/substrafl_doc/substrafl_overview.rst

```diff
@@ -102,3 +102,32 @@ Each round represents one iteration of the training loop in the federated settin
 * The weight updates are aggregated by the aggregator organization.
 * The aggregated organization send the aggregated updates to the training organizations.
 * The training organizations update their model with the aggregated updates.
+
+Centralized strategy - workflow
+--------------------------------
+
+The workflow of a centralised strategy, unless specified otherwise, is as follows:
+
+- initialisation round: one train task on each train organization
+- then for each round: one aggregate task on the central organization then one train task on each train organization
+
+Steps of an aggregate task:
+
+- Calculate the common shared state from the previous train tasks shared state.
+
+Steps of a train task:
+
+- If there is an aggregate task before: update the model parameters with the shared state
+- Train the model on the local data
+- Calculate the shared state update
```

Comment:
Here we speak of local / shared state, but what the user can actually download is trunk and head. Should we add a warning above saying that shared=trunk, local=head and that will change shoon?

Reply:
I would not do that, as the 'download model' function hides these concepts

Reply:
What do you mean?
The download model does expose the head/trunk concepts --> https://docs.substra.org/en/latest/documentation/references/sdk.html#download-head-model-from-composite-traintuple

Reply:
I mean the download model function in substrafl, to download the proper FL model

Reply:
ah yes

Reply:
Thanks!